openff/interchange/_tests/unit_tests/interop/lammps/export/test_lammps.py

```diff
@@ -1,6 +1,10 @@
+from pathlib import Path
+from typing import Any, Dict, Type
+
 import numpy
 import pytest
-from openff.toolkit import Topology, unit
+from openff.toolkit import Molecule, Topology, unit
+from openff.toolkit.typing.engines.smirnoff import ForceField
```

Comment:
```suggestion
from openff.toolkit import Molecule, Topology, unit, ForceField
```

Reply:
ForceField import from `...smirnoff` was because the direct import from openff.toolkit raised an import error on my end when attempting to import TestLammps. I'll consolidate these and see if the issue persists, or if the test remains runable

Reply:
The quick imports should be available in any recent toolkit versions, starting somewhere around 0.11 or 0.12

Reply:
If you can't `from openff.toolkit import ForceField` that's an issue to me because it's a huge surprise, not because of the style